pkg/util/istiomultierror/util.go

```diff
@@ -0,0 +1,47 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package istiomultierror
```

Comment:
nit: Following the conventions of other packages in util - may be we should move it to multierror/multierror.go and places where we import we can qualify with `istiomultierror` to make it clear

Reply:
I was a bit worried about making *every* usage need to qualify it

Reply:
Ok. I am just thinking it is better if we follow the existing conventions in this folder - but up to you. Can we change util.go ->  <istio>multierror.go?